### PR TITLE
Create alert rules for integration tests

### PR DESCRIPTION
**Pull request by suleymanakbas91**
**Description**

Changes proposed in this pull request:

- Create alert rules on Prometheus for integration test failures 
- Add Slack configurations to Alertmanager
- Update documentation

**Related issue(s)**

See #15

### CODE REVIEW

File: docs/prow/prow-monitoring.md [hunk 1]
```diff
@@ -30,7 +38,7 @@ Follow these steps:
 4. Install the monitoring chart:
 
    ```bash
-   helm install --name {releaseName} --namespace {namespaceName} resources/monitoring
+   helm install --name {releaseName} --namespace {namespaceName} resources/monitoring -f resources/monitoring/values.yaml
```
Comment (Demonsthere) [inline docs/prow/prow-monitoring.md]: Those are the default chart values right? We shouldn't need to specify them

**Comment by michal-hudy** [comment]
@suleymanakbas91 Can you post how notification will look in slack?

**Comment by suleymanakbas91** [comment]
@michal-hudy 
screenshot 2019-01-03 at 15 54 57

File: prow/cluster/resources/monitoring/templates/prometheus-rule.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+metadata:
+  name: prometheus-prow-job-failure-rules
+  labels:
+    app: prometheus-operator
+{{ include "prometheus-operator.labels" . | indent 4 }}
+spec:
+  groups:
+  - name: ./prow.rules
+    rules:
+    - alert: KymaIntegration
```
Comment (michal-hudy) [inline prow/cluster/resources/monitoring/templates/prometheus-rule.yaml]: I think we should monitor all `postsubmit` and `periodics` jobs. Not only integration.

File: prow/cluster/resources/monitoring/values.yaml [hunk 3]
```diff
@@ -2,6 +2,32 @@ grafana:
...
+    config:
+      global:
+        resolve_timeout: 5m
+      receivers:
+      - name: "null"
+      - name: "slack"
+        slack_configs:
+        - channel: <SLACK_CHANNEL>
+          send_resolved: true
+          api_url: <SLACK_URL>
```
Comment (kazydek) [inline prow/cluster/resources/monitoring/values.yaml]:
```suggestion
          api_url: {SLACK_URL}
```

File: prow/cluster/resources/monitoring/values.yaml [hunk 4]
```diff
@@ -2,6 +2,32 @@ grafana:
...
 prometheus-operator:
+  alertmanager:
+    config:
+      global:
+        resolve_timeout: 5m
+      receivers:
+      - name: "null"
+      - name: "slack"
+        slack_configs:
+        - channel: <SLACK_CHANNEL>
```
Comment (kazydek) [inline prow/cluster/resources/monitoring/values.yaml]:
```suggestion
        - channel: {SLACK_CHANNEL}
```

File: prow/cluster/resources/monitoring/values.yaml [hunk 5]
```diff
@@ -2,6 +2,32 @@ grafana:
...
+          send_resolved: true
+          api_url: <SLACK_URL>
+          text: "<!channel> \ndescription: {{ .CommonAnnotations.description }}"
```
Comment (kazydek) [inline prow/cluster/resources/monitoring/values.yaml]:
```suggestion
          text: "{!channel} \ndescription: {{ .CommonAnnotations.description }}"
```

File: docs/prow/prow-monitoring.md [hunk 6]
```diff
@@ -9,6 +9,14 @@ Install the following tools:
 - Helm v2.11.0
 - kubectl
 
+## Configure Slack for failure notifications
+
+Follow these steps:
+
+1. Create a Slack channel and create an [Incoming Webhook](https://api.slack.com/incoming-webhooks) for this channel. Copy the resulting Webhook URL.
+
+2. Replace **<SLACK_URL>** on [values.yaml](./../../prow/cluster/resources/monitoring/values.yaml) with the Weebhook URL you obtaied and replace **<SLACK_CHANNEL>** with the channel name.
```
Comment (kazydek) [inline docs/prow/prow-monitoring.md]:
```suggestion
2. Replace **{SLACK_URL}** in [values.yaml](./../../prow/cluster/resources/monitoring/values.yaml) with the Weebhook URL and **{SLACK_CHANNEL}** with the channel name.
```

**Pull request merged**